force_wfmanager/ui/setup/workflow_tree.py

```diff
@@ -746,11 +746,11 @@ def verify_tree(self, errors, start_view=None):
 
             # Further checks for UI objects not relating to an object in the
             # force_bdss Workflow model
-            err_subject_type = type(verifier_error.subject)
 
             # For errors where the subject is an Input/OutputSlotInfo object,
             # check if this is an attribute of the (DataSource) model
-            if err_subject_type in [InputSlotInfo, OutputSlotInfo]:
+            if (isinstance(verifier_error.subject, InputSlotInfo)
```

Comment:
With `isinstance` you can pass it a tuple of classes like: 
`isinstance(verifier_error.subject, (InputSlotInfo, OutputSlotInfo))`